docs/reference/skin.md

```diff
@@ -0,0 +1,290 @@
+## Skin
+
+Derived from [Device](device.md).
+
+```
+Skin {
+  SFVec3f      translation     0 0 0
+  SFRotation   rotation        0 1 0 0
+  SFVec3f      scale           1 1 1
+  SFString     name            "skin"
+  SFString     modelUrl        ""
+  MFNode       appearance      []
+  MFNode       bones           []
+  SFBool       castShadows     TRUE
+  SFFloat      translationStep 0.01
+  SFFloat      rotationStep    0.261799387
+}
+```
+
+### Description
+
+The [Skin](#skin) node can be used to simulate soft mesh animation for example of a human or an animal.
+The skin mesh is imported from a mesh file specified by the `model` name but in order to be animated it has to be attached to a skeleton so that the rotation of the skeleton joints results in appropriate deformation of the skin mesh.
+
+The [Skin](#skin) provides two alternative ways to define a skeleton.
+The first method consists in providing the skeleton in the mesh file.
+The second method consists in listing the [Solid](solid.md) nodes corresponding to the mesh bones using the `bones` field.
+If in the first case the resulting object animation will be purely graphical.
+In the second case, when linking the [Skin](skin.md) to an existing Webots skeleton made of [Solid](solid.md) and [Joint](joint.md) nodes, it is possible to animate a dynamic object.
+
+The supported format for the skin mesh and skeleton models is [FBX](https://en.wikipedia.org/wiki/FBX) and the model be generated using a 3D modeling software.
+Other tools can be used for generating the characters to animate, for example human characters provided in the "[skin\_animated\_humans]({{ url.github_tree }}/projects/humans/skin_animated_humans/worlds/skin_animated_humans.wbt)" sample are generated using [MakeHuman](http://www.makehuman.org/).
```

Comment:
```suggestion
Other tools can be used for generating the characters to animate, for example human characters provided in the "[skin\_animated\_humans]({{ url.github_tree }}/projects/humans/skin_animated_humans/worlds/skin_animated_humans.wbt)" sample are generated using [MakeHuman](http://www.makehumancommunity.org/).
```

When I go on http://www.makehuman.org/ it says that the website has moved to this new address